tests/integration/test_quantiles.py

```diff
@@ -0,0 +1,434 @@
+# Copyright 2022 NVIDIA Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+
+import numpy as np
+import pytest
+from legate.core import LEGATE_MAX_DIM
+from utils.comparisons import allclose
+
+import cunumeric as cu
```

Comment:
@bryevdv has requested that we are consistent in using `import cunumeric as num` in tests going forward.

Reply:
Done.

Reply:
It was actually `cunumeric` that we needed to change to be imported as `num`, not `numpy`. I've pushed the corresponding changes.